### PR TITLE
Gui buttons

### LINKED ISSUE #6: Extra GUI buttons
The GUI should also contain the following:
- `Previous Track` button
- `Playback speed` - to play the song at different speeds
- `Shuffle` button/check box - when a user clicks `Next song` a random song is played
- `Repeat` button/check box - when the track list has finished playing, start from the top

All of the above should also be implemented in the backend.  
At the moment the repeat functionality is provided but there is no way of stopping the player from repeating the list over and over again.

### PR DESCRIPTION
**Pull request by gabi-250**
This branch introduces the following functionalities:
* a shuffle option for the playlist
* the possibility to change the playback speed

This closes #6

### CODE REVIEW

File: openmedia/gui/controlbox.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from .tools.iconhelp import get_button_image
 from .tools.iconhelp import get_name
 from ..player import player
-
+from . import WINDOW_WIDTH, WINDOW_HEIGHT
```
Comment (rbartlensky) [inline openmedia/gui/controlbox.py]: Add an extra line for PEP8 compliance.

File: openmedia/gui/mainwindow.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from openmedia.observable.observable import Observer
 from .progressbar import ProgressBar
 from .controlbox import ControlBox
-
+from . import WINDOW_WIDTH, WINDOW_HEIGHT
```
Comment (rbartlensky) [inline openmedia/gui/mainwindow.py]: Add new line for PEP8 compliance.

File: openmedia/player/player.py [hunk 3]
```diff
@@ -144,7 +179,30 @@ def add(self, track_path):
         return False
 
     def get_current_second(self):
-        return self.pipeline.query_position(3)[1] / 10**9
+        return self.pipeline.query_position(Gst.Format.TIME)[1] / 10**9
```
Comment (rbartlensky) [inline openmedia/player/player.py]: I think we should use `Gst.SECOND` everywhere we used `10**9` just to make the code clearer.

File: openmedia/gui/controlbox.py [hunk 4]
```diff
@@ -74,14 +85,35 @@ def _play_next(self, widget):
         if self.playlist_box.is_visible():
             self.playlist_box.show_all()
 
+    def _play_previous(self, widget):
+        Player.instance().play_previous()
+        if self.playlist_box.is_visible():
```
Comment (rbartlensky) [inline openmedia/gui/controlbox.py]: I am not sure what this line is supposed to do.

**Review by rbartlensky — CHANGES_REQUESTED** [review]

File: openmedia/gui/controlbox.py (hunk 4 above)
Comment (gabi-250) [reply openmedia/gui/controlbox.py]: Yep, it seems to be unnecessary. I'll remove it from `_play_previous` and `_play_next`.

**Review by rbartlensky — APPROVED** [review]

**Comment by rbartlensky** [comment]
You can squash now.

**Pull request merged**